README.md

```diff
@@ -14,169 +12,202 @@
+logs, pre-aggregating the logs and ingesting into the built-in in-memory columnar store (where the data is stored both
+in 'exact' form as well as a stratified sample).
 We showcase the following aspects of this unified cluster:
-- Simplicity of using SQL or the [DataFrame API](http://spark.apache.org/docs/latest/sql-programming-guide.html#dataframes) to model streams in spark.
-- The use of SQL/SchemaDStream API (as continuous queries) to pre-aggregate AdImpression logs (it is faster and much more convenient to incorporate more complex analytics, rather than using map-reduce).
-- Demonstrate storing the pre-aggregated logs into the SnappyData columnar store with high efficiency. While the store itself provides a rich set of features like hybrid row+column store, eager replication, WAN replicas, HA, choice of memory-only, HDFS, native disk persistence, eviction, etc we only work with a column table in this simple example.
-- Run OLAP queries from any SQL client both on the full data set as well as sampled data (showcasing sub-second interactive query speeds). The stratified sample allows us to manage an infinitely growing data set at a fraction of the cost otherwise required.
+- Simplicity of using the [DataFrame API](http://spark.apache.org/docs/latest/sql-programming-guide.html#dataframes) to
+model streams in spark.
+- The use of Structured Streaming API to pre-aggregate AdImpression logs (it is faster and much more convenient to
+incorporate more complex analytics, rather than using map-reduce).
+- Demonstrate storing the pre-aggregated logs into the SnappyData columnar store with high efficiency. While the store
+itself provides a rich set of features like hybrid row+column store, eager replication, WAN replicas, HA, choice of memory-only, HDFS, native disk persistence, eviction, etc we only work with a column table in this simple example.
+- Run OLAP queries from any SQL client both on the full data set as well as sampled data (showcasing sub-second
+interactive query speeds). The stratified sample allows us to manage an infinitely growing data set at a fraction of the
+cost otherwise required.
 
 ### Ad Impression Analytics use case
-We borrow our use case implementation from this [blog](https://chimpler.wordpress.com/2014/07/01/implementing-a-real-time-data-pipeline-with-spark-streaming/) - We more or less use the same data structure and aggregation logic and we have adapted this code to showcase the SnappyData programming model extensions to Spark. We retain the native Spark example for comparison.
+We borrow our use case implementation from this [blog](https://chimpler.wordpress.com/2014/07/01/implementing-a-real-time-data-pipeline-with-spark-streaming/)
+\- We more or less use the same data structure and aggregation logic and we have adapted this code to showcase the
+SnappyData programming model extensions to Spark. We have also kept the native Spark example using structured streaming
```

Comment:
"We have also kept..." - Should be "We also run structured streaming example using native Spark".

Reply:
I have added link to spark example code from the repo. That will make it more clear.